prefect_gcp/cloud_storage.py

```diff
@@ -436,7 +436,7 @@ def example_cloud_storage_copy_blob_flow():
     logger = get_run_logger()
     logger.info(
         "Copying blob named %s from the %s bucket to the %s bucket",
-        source_blob,
+        dest_blob or source_blob,
```

Comment:
I think this should still be just `source_blob`. If you want to expand the log message to include what the name of the destination blob will be, I think that makes sense.
```suggestion
        source_blob,
```

Reply:
You're 100% correct! I misread the log.